scheduler/pkg/agent/repository/triton/triton_test.go

```diff
@@ -257,3 +272,73 @@ output [
 		})
 	}
 }
+
+func TestCopyNonConfigFilesToModelRepo(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	type test struct {
+		name                 string
+		rcloneFiles          []string
+		rcloneFolders        []string
+		repoFoldersToCreate  []string
+		expectedRepoPaths    []string
+		notExpectedRepoPaths []string
+	}
+
+	tests := []test{
+		{
+			name:              "files",
+			rcloneFiles:       []string{"t"},
+			expectedRepoPaths: []string{"t"},
+		},
+		{
+			name:                 "files and folders",
+			rcloneFiles:          []string{"t"},
+			rcloneFolders:        []string{"folder"},
+			expectedRepoPaths:    []string{"t"},
+			notExpectedRepoPaths: []string{"folder"},
+		},
+		{
+			name:                 "files and folders with config.pbtxt and existing folders in repo",
+			rcloneFiles:          []string{"t", "config.pbtxt"},
```

Comment:
why there is a test case with `config.pbtxt` in  `TestCopyNonConfigFilesToModelRepo`?

Reply:
Want to check its not copied